app/styles/app.scss

```diff
@@ -13,6 +13,7 @@
 @import 'components/x-card';
 
 $primary-color-palette: $color-orange;
+$locked-breakpoint: 5000px;
```

Comment:
This was the variable ember-paper is using for the min-width on showing the sidenav. I set it really high to effectively force it hidden all the time. We could set this to something more reasonable, like 1900px or something, so people with big monitors aren't inconvenienced.